cg/server/api.py

```diff
@@ -467,6 +470,17 @@ def get_application_pipeline_limitations(tag: str):
     return jsonify([limitation.to_dict() for limitation in application_limitations])
 
 
+@BLUEPRINT.route("/orders")
+def get_orders():
+    """Return the latest orders."""
+    orders_request: OrdersRequest = OrdersRequest.model_validate(request.args)
+    order_service = OrderService(db)
```

Comment:
If possible, I'd initialize this service in the app context and retrieve it like
```python
order_service: OrderService = current_app.extensions.get("order_service")
``` 
That would require setting it up like this in `app.py`:
```python
# Services
order_service = OrderService(store)

# Register services
app.extensions["order_service"] = analysis_service
```

Or in some other similar way, I have not found a great way for injecting services yet in Python but I think this works for now. Here is how it is [done in Trailblazer](https://github.com/Clinical-Genomics/trailblazer/blob/0607d626f8e85e07398113db0fd50c4845db1d52/trailblazer/server/app.py#L29).

Reply:
@seallard There is no such pattern in cg though, and the store is never even mentioned in app.py. If anything, I could initialize it in ext.py where the "db" variable is instantiated. That does not sound terribly intuitive to me though.

Reply:
Let skip it for now!